flink-libraries/flink-table/src/test/java/org/apache/flink/api/java/table/test/StringExpressionsITCase.java

```diff
@@ -40,46 +40,6 @@ public StringExpressionsITCase(TestExecutionMode mode) {
 		super(mode);
 	}
 
-	@Test(expected = CodeGenException.class)
```

Comment:
Yes, you are right, testing at least one function end-to-end makes sense. I will add them again.